contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -5033,3 +5034,21 @@ get_bbf_pivot_tuplestore(RawStmt 	*sql,
 
 	return tupstore;
 }
+
+/*
+ * bbf_xmlexist_helper()
+ * Returns a bit that represents one of the following conditions:
+ * 	1, representing True, if the XPATH expression returns a nonempty result. That is, it returns at least one XML node.
+ * 	0, representing False, if it returns an empty result.
+ * 	NULL if the xml data type instance against which the query was executed contains NULL.
+ */
+Datum
+bbf_xmlexist_helper(PG_FUNCTION_ARGS)
```

Comment:
Blocking: Why cannot we check the GUC value in SQL function itself?

Reply:
removed the C function and added check in SQL function itself.